### PR TITLE
[Finished] Cleanup & rpm stuff

### PR DESCRIPTION
**Pull request by z3ntu**
- [x] Makefile: Remove fedora scripts
- [x] Misc: Remove upstart file
- [x] RPM: Add rpm (fedora/opensuse) spec
- [x] Daemon&pylib: Replace manual python install with python setuputils
- [x] Bumpversion: Add new version strings to bumpversion.cfg

### CODE REVIEW

File: rpm/razer-drivers.spec [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+Name: razer-drivers
+Version: 1.1.3
+Release: 1%{?dist}
+Summary: Razer drivers for Linux
+
+License: GPL-2.0
+URL: https://github.com/terrycain/razer-drivers
+
+#Source0: https://github.com/terrycain/razer-drivers/archive/v%{version}.tar.gz
```
Comment (terricain) [inline rpm/razer-drivers.spec]: Can we get that commented out version Source0 line working?

File: rpm/razer-drivers.spec [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+%package -n razer-daemon
+Summary: Razer Service package
+Group: System Environment/Daemons
+BuildRequires: python3-devel
+BuildRequires: python3-setuptools
+Requires: razer-kernel-modules-dkms
+Requires: python3
+# Thanks openeuse for this great package name...
+%if 0%{?suse_version}
+Requires: dbus-1-python3
```
Comment (terricain) [inline rpm/razer-drivers.spec:50]: Now that is a quuality package name

File: rpm/razer-drivers.spec [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+%install
+rm -rf $RPM_BUILD_ROOT
+# setup_dkms & udev_install -> razer-kernel-modules-dkms
+# daemon_install -> razer_daemon
+# python_library_install -> python3-razer
+make DESTDIR=$RPM_BUILD_ROOT setup_dkms udev_install daemon_install python_library_install
+mv $RPM_BUILD_ROOT/lib/udev $RPM_BUILD_ROOT/usr/lib/udev
+# Soon not neccessary anymore :)
+rm -rf $RPM_BUILD_ROOT/%{_datadir}/upstart/
```
Comment (terricain) [inline rpm/razer-drivers.spec]: PURGE TEH UPSTART :wink:

**Review by terricain — CHANGES_REQUESTED** [review]
Nice

**Review by terricain — APPROVED** [review]
I still need to test the building of the rpms, but looks good.

**Comment by z3ntu** [comment]
Please just try the spec file WITH this pr, otherwise some files are "unpackaged" and on a sidenote the build will just fail on opensuse tumbleweed 32bit ^^

**Comment by z3ntu** [comment]
(something like this should be added to the website after this is merged, I can also transfer the repo to terry so he has full control over it and can revoke rights from me if needed 😉  )

## How to install on Fedora 25
```
sudo dnf config-manager --add-repo http://download.opensuse.org/repositories/home:z3ntu/Fedora_25/home:z3ntu.repo
sudo dnf install razer-drivers kernel-devel # be sure to include the kernel-devel package here, because Fedora has a problem with that package for at least 1.5 years
```

## How to install on OpenSUSE
Follow the instructions at https://software.opensuse.org/download.html?project=home%3Az3ntu&package=razer-drivers .

**Comment by z3ntu** [comment]
Rebased (doesn't matter but still) and from new branch in #131

**Pull request closed without merging**